frontend/src/component/admin/groups/GroupForm/GroupFormUsersSelect/GroupFormUsersSelect.tsx

```diff
@@ -67,6 +77,27 @@ export const GroupFormUsersSelect: VFC<IGroupFormUsersSelectProps> = ({
     setUsers,
 }) => {
     const { users: usersAll } = useUsers();
+    const { serviceAccounts } = useServiceAccounts();
+
+    const options = [
+        ...usersAll
+            .map((user: IUser) => ({ ...user, type: 'USERS' }))
+            .sort((a: IUser, b: IUser) => {
+                const aName = a.name || a.username || '';
+                const bName = b.name || b.username || '';
+                return aName.localeCompare(bName);
+            }),
+        ...serviceAccounts
+            .map((serviceAccount: IServiceAccount) => ({
+                ...serviceAccount,
+                type: 'SERVICE ACCOUNTS',
+            }))
+            .sort((a, b) => {
+                const aName = a.name || a.username || '';
+                const bName = b.name || b.username || '';
+                return aName.localeCompare(bName);
+            }),
+    ];
```

Comment:
Would this have the same result? Feel free to dismiss it

```suggestion
    const options = [
        ...usersAll
            .map((user: IUser) => ({ ...user, type: 'USERS' } as UserOption)),
        ...serviceAccounts
            .map((serviceAccount: IServiceAccount) => ({
                ...serviceAccount,
                type: 'SERVICE ACCOUNTS',
            } as UserOption)),
    ].sort((a: UserOption, b: UserOption) => {
                const aName = a.name || a.username || '';
                const bName = b.name || b.username || '';
                return aName.localeCompare(bName);
    });
```

Reply:
I believe it wouldn't, as according to the current implementation each option should be sorted inside its own category. 

Here's what happens with the suggested code:

![image](https://user-images.githubusercontent.com/14320932/213160370-79c315b9-06a1-42cf-a762-8910e086aebd.png)

Reply:
I thought that sorting in "sections" would create some sort of sub-ordering. Of course we can first order by `type` and then by name, but I think your code would be clearer and easy to maintain